bucket.go

```diff
@@ -123,8 +123,11 @@ func (b *Bucket) Bucket(name []byte) *Bucket {
 func (b *Bucket) openBucket(value []byte) *Bucket {
 	var child = newBucket(b.tx)
 
-	// Unaligned pointer access requires a copy to be made.
-	const unalignedMask = unsafe.Alignof(uintptr(0)) - 1
+	// Unaligned access requires a copy to be made.
+	const unalignedMask = unsafe.Alignof(struct {
+		bucket
+		page
+	}{}) - 1
```

Comment:
I don't have any 32-bit hardware available (i386, armv7, etc) so if someone could test whether this still passes with `-gcflags=all=-d=checkptr` on these architectures I'd love to know.  My concern is that the allocation for the copied byte slice below will be pointer aligned (4 bytes), and not aligned for the 64-bit fields of these structs (8 bytes).